tests/network-tests/src/flows/clis/createChannel.ts

```diff
@@ -48,6 +50,22 @@ export default async function createChannel({ api, query }: FlowProps): Promise<
     memberId.toString(),
   ])
 
+  const expectedChannelRewardAccount = api
+    .createType(
+      'AccountId32',
+      u8aFixLength(
+        u8aConcat(
+          'modl',
+          'mContent',
+          createType('Bytes', 'CHANNEL').toU8a(false),
+          createType('u64', channelId).toU8a()
+        ),
+        32 * 8,
+        true
+      )
+    )
+    .toString()
```

Comment:
Just curious how you found out what the parameters to `u8aConcat` should be? I mean, great work because I don't know where I would start to look for them.